### ISSUE TITLE
[RTL] HiddenColumns

### ISSUE THREAD

**Issue by Siemienik** [handsontable/handsontable#8829]
### Description

For RTL, currently, the hidden column indicators are displayed on the opposite sides of headers. It should be adjusted to displaying on the proper site. 

### How to reproduce

1. Use this [configuration](https://github.com/Siemienik/handsontable-storybooks/blob/63a103e4b8dace9d3db2162da372382533c2ddb7/vanilla/configurations.js#L793-L805) 
```js
{
    licenseKey: 'non-commercial-and-evaluation',
    data: Handsontable.helper.createSpreadsheetData(5, 12),
    colHeaders: true,
    rowHeaders: true,
    height:'auto',
    // individually add column hiding context menu items
    contextMenu: [`hidden_columns_show`, `hidden_columns_hide`],
    hiddenColumns: {
        columns: [3, 5, 9],
        indicators: true
    }
}
```
2. result:
<img width="572" alt="Screenshot 2021-10-05 at 11 46 59" src="https://user-images.githubusercontent.com/2893099/136001919-58a5a8e0-8462-412b-974f-8eb0504d9e1f.png">
3. Indicator should be between CE, EG, and IK

**Comment by aninde** [comment]
Resolved in the v12.0.0 https://jsfiddle.net/jz6amgx3/

**Comment by AMBudnik** [comment]
Closed with https://github.com/handsontable/handsontable/issues/8760 (RTL Support), published in Handsontable v12.

For more information please check out our Release Notes - https://handsontable.com/docs/release-notes/#_12-0-0
